README.md

```diff
@@ -63,6 +65,7 @@ See [Testing Farm docs](https://docs.testing-farm.io) for more information on su
 | `debug` | Print debug logs when working with testing farm | true |
 | `update_pull_request_status` | Action will update pull request status. Default: true | true |
 | `environment_settings` | Pass custom settings to the test environment. Default: {} | empty |
+| `pr_head_sha` | SHA of the latest commit in PR. Used for communication with GitHub API. | $(git rev-parse HEAD) |
 
 ## Example
```

Comment:
Does it make sense to add the following to the example?
```yml
update_pull_request_status: "true"
pr_head_sha: ${{ github.event.pull_request.head.sha }}
```

Reply:
Hmm, the existing example shows triggering action on `issue_comment`. Do  you somewhere have some working example, that action triggered by `issue_comment` has access to `${{ github.event.pull_request.head.sha }}` variable? I did not try that by myself.

Or we could probably write second example triggered by some other trigger than `issue_comment`. But that example should also show that only authorized users can trigger tests on Testing Farm infrastructure.

Reply:
@r0x0d, do you know if the `issue_comment` trigger makes `github.event.pull_request.head.sha` available?

In the example there's a condition to make sure the issue that someone commented on is a PR.
```yaml
if: |
  github.event.issue.pull_request
```

Reply:
I haven't tested that in a while, but I believe it won't work… I will try it in my test repo

Reply:
To be honest, I would say it's super okay to have the `$(git rev-parse HEAD)` as an example there. Most people would already know how to get their SHA in different ways other than `git rev-parse`, I understand it as a more general way of saying "Hey, this is the input you need to pass to us, an actual SHA. Don't care if it is the HEAD SHA or the previous one an so on..."

But that just my opinion.

Reply:
I would also leave it as it is for now.
If you have some working example of SHA inserted as an input you can always create a PR to the `testing-farm-github-action` repository with it :).

What do you say, @bocekm?

Reply:
Ok, let's leave the example as it. Thanks for looking into it.